src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -114,6 +114,12 @@
 
     <!-- All Microsoft.NET.Sdk Exe projects, except WPF and WinForms exes, can be published to app service -->
     <ProjectCapability Include="AppServicePublish" Condition="'$(OutputType)' == 'Exe'" />
+
+    <!-- Capabilities for WPF and WinForms -->
+    <ProjectCapability Include="WPF" Condition="'$(UseWPF)' == 'true'" />
+    <ProjectCapability Include="WindowsForms" Condition="'$(UseWindowsForms)' == 'true'" />
+    <!-- XBAP is not supported in .NET Core -->
+    <ProjectCapability Include="XBAP" Condition="'$(HostInBrowser)' == 'true' And '$(TargetFrameworkIdentifier)' == '.NETFramework'" />
```

Comment:
Can we remove this until absolutely needed? We'll have nothing light up on this yet.